api/src/services/accountsService.js

```diff
@@ -0,0 +1,61 @@
+const { blockchainNames, tokenTypes } = require("../config/blockchains");
+const { blockchains } = require("../config");
+const cereNetworkService = require('./cereNetworkService');
+const ethNetworkService = require('./ethNetworkService');
+
+async function  get() {
+  const promises  = [];
+  blockchains.forEach(blockchain => {
+    blockchain.networks.forEach(network => {
+      network.accounts && network.accounts.forEach(account => {
+        const promise = async () => {
+          let balance;
+          switch (blockchain.name) {
+            case blockchainNames.CERE:
+              switch(account.type) {
+                case tokenTypes.NATIVE:
+                  balance = await cereNetworkService.getBalance(network.name, account.address);
+                  break;
+              }
+              break;
```

Comment:
Let's also add warn logs when the tokenType is unknown to all blockchains

Reply:
resolved